src/pages/en/reference/api-reference.md

```diff
@@ -274,7 +274,7 @@ export async function getStaticPaths() {
   return [
     { params: { id: '1' } },
     { params: { id: '2' } },
-    { params: { id:  3 } }
+    { params: { id:  3  }  }  // Can be a number too! Only strings and numbers are supported values.
```

Comment:
```suggestion
    { params: { id:  3  }  }  // Can be a number too!
```
OK, I'm going to make an executive decision and say that after testing at a few different screen sizes, I dislike the longer comment making an otherwise narrow code sample need to be scrollable!

So, I suggest we just point out that it can be a number. And, @shaunchander if you can also "update branch" to sync up with main, then I'm happy to merge!